src/app/clusters/microwave-oven-control-server/microwave-oven-control-server.h

```diff
@@ -125,28 +134,56 @@ class Delegate
     virtual ~Delegate() = default;
 
     /**
-     *   @brief Handle Command Callback in application: SetCookingParameters
-     *   @return Returns the Interaction Model status code which was user determined in the business logic
-     *   @param  cookMode: the user defined modes which from the Microwave Oven Mode application level
-     *   @param  cookTime: the input cook time value
-     *   @param  powerSetting: the input power setting value
+     *   @brief Handle Command Callback in application: SetCookingParameters.
+     *   @return Returns the Interaction Model status code which was user determined in the business logic.
+     *   @param  cookMode: the user defined modes which from the Microwave Oven Mode application level.
+     *   @param  cookTime: the input cook time value.
+     *   @param  startAfterSetting: the flag to indicate if start for the operation after handling the command.
+     *   @param  feature: featureMap bits value of Microwave Oven Control.
+     *   @param  powerSetting: the input power setting value.
+     *   @param  wattSettingIndex: the input watts setting index.
+     *   Note: powerSetting and wattSettingIndex must be mutually exclusive, depends on the feature value.
+     *   Caller should check if the feature has included the corresponding feature value at the application level.
+     *   If using power as number, wattSettingIndex will be set to NullOptional.
+     *   If using power in watts, powerSetting will be set to NullOptional.
```

Comment:
Much better, thank you.

I don't know why we need features as an argument here anymore, but ok.

Reply:
Will remove the feature argument. SDK user does know what the feature has set  in the application, there is no need to check the feature bits.